### DISCUSSION TITLE
Install on unraid help

### DISCUSSION THREAD

**Discussion by wildfirebill** [a1ex4/ownfoil#78] [Q&A]
Adding an update, at work right now but came back to check on this and went to the repo and noticed something different in the compose.yml file I used the compose file from docker hub and added :v2 to the end. I see a extra setting that isn't in my configuration. When I get off work I will update it.

I am trying to setup for my unraid server but nothing is going on. It has been running for 2 hours. I seen a post saying it took 30 minutes to get started so I waited. I am not very experienced with unraid or docker to be honest if anyone can help me out I would appreciate it

![ownfoil1](https://github.com/a1ex4/ownfoil/assets/8473557/e412b8c3-908d-4225-ba21-ab4088d67779)
![ownfoil2](https://github.com/a1ex4/ownfoil/assets/8473557/a0e86aef-7aef-4745-a50a-da0f8f86b8ad)
![ownfoil3](https://github.com/a1ex4/ownfoil/assets/8473557/bb451abd-89c1-40cc-a82e-d9ac6c19a9a2)
![ownfoil4](https://github.com/a1ex4/ownfoil/assets/8473557/2d72d648-d897-400a-b831-d53cc2545721)
![ownfoil5](https://github.com/a1ex4/ownfoil/assets/8473557/48fdc289-456f-424f-a878-25d1e6978d02)

**Comment by wildfirebill** [comment]
Alright so all that is fixed. only thing that is not working is getting it to work with NPM so I can connect to it while outside the local network.

**Comment by Sev7nrayne** [comment]
wan:port <--router--> internal ip:port

setup port forwarding in router
